R/derive_vars_aage.R

```diff
@@ -231,66 +232,23 @@ NULL
 #' @rdname derive_var_agegr_fda
 #'
 #' @keywords deprecated
+#' @family deprecated
 #'
 #' @export
 derive_var_agegr_fda <- function(dataset, age_var, age_unit = NULL, new_var) {
-  deprecate_warn("0.8.0", "derive_var_agegr_fda()", details = "Please create a user defined function instead.")
-
-  age_var <- assert_symbol(enexpr(age_var))
-  new_var <- assert_symbol(enexpr(new_var))
-  warn_if_vars_exist(dataset, as_name(new_var))
-
-  ds <- derive_var_age_years(dataset, !!age_var, age_unit, new_var = temp_age)
-
-  out <- ds %>%
-    mutate(
-      !!new_var := cut(
-        x = temp_age,
-        breaks = c(0, 18, 65, Inf),
-        labels = c("<18", "18-64", ">=65"),
-        include.lowest = TRUE,
-        right = FALSE
-      )
-    ) %>%
-    select(-temp_age)
-
-  if (anyNA(dplyr::pull(out, !!new_var))) {
-    out <- mutate(out, !!new_var := addNA(!!new_var))
-  }
-  out
+  deprecate_stop("0.10.0", "derive_var_agegr_fda()",
+    details = "Please create a user defined function instead."
```

Comment:
Perhaps include this statement in the roxygen documentation too?
![image](https://user-images.githubusercontent.com/43018762/219120541-a56e3df2-b960-4571-a2bb-eb38e8fdffa3.png)